Jenkinsfile

```diff
@@ -168,6 +168,24 @@ try {
         }
       }
     },
+    'CPU: Clang': {
+      node('mxnetlinux-cpu') {
+        ws('workspace/build-cpu-clang') {
+          init_git()
+          def flag = """ \
+            USE_PROFILER=1                \
+            USE_CPP_PACKAGE=1             \
+            USE_BLAS=openblas             \
+            USE_OPENMP=0                  \
```

Comment:
I guess `USE_OPENMP` is set to 0 because it doesn't work on the version of clang that ships with OSX. However, openmp is supported in other new clang versions, and are also available on OSX through the llvm package on homebrew. Given that the test runs on linux I think openmp should be tested too.

Reply:
Yes, correct about disabling USE_OPENMP.  The goal of this job is to protect against accidental broken builds on OSX by using a similar build chain and flags on linux.

I think you do have a good point though, I can pretty easily create another job with a modern clang that has USE_OPENMP turned on.  IMO the extra environment would be valuable to test, and shouldn't use too many resources relative to other CI jobs.